Buildings/Resources/src/ThermalZones/Detailed/Validation/BESTEST/simulateAndPlot.py

```diff
@@ -0,0 +1,988 @@
+                        'max_temperature', 'min_temperature', 'ave_temperature']
+    barPlot_yLabel = ['Annual Heating Load [MWh]', 'Annual Cooling Load [MWh]', 'Peak Heating Load [kW]', 'Peak Cooling Load [kW]', \
+                      'Maximum Temperature [degC]', 'Minimum Temperature [degC]', 'Average Temperature [degC]']
+    barPlot_yMin = [0, 0, 0, 0, 0, -25, 0]
+    barPlot_yMax = [8, 9, 8, 8, 80, 10, 35]
+    line_variable = ['FF_temperature_600FF_Jan4', 'FF_temperature_900FF_Jan4', 'FF_temperature_650FF_Jul27', \
+                     'FF_temperature_950FF_Jul27', 'hourly_load_600_Jan4', 'hourly_load_900_Jan4', 'bin_temperature_900FF']
+    line_xLable = ['Hour of Day (Case600FF, Jan4)', 'Hour of Day (Case900FF, Jan4)', 'Hour of Day (Case650FF, Jul27)', \
+                   'Hour of Day (Case950FF, Jul27)', 'Hour of Day (Case600, Jan4)', 'Hour of Day (Case900, Jan4)', 'Temperature Bins [degC] (Case900FF)']
+    line_yLable = ['Temperature [degC]', 'Temperature [degC]', 'Temperature [degC]', 'Temperature [degC]', \
+                   'Heating or Cooling Load [kW]', 'Heating or Cooling Load [kW]', 'Number of Occurrences']
+    line_xMin = [1, 1, 1, 1, 1, 1, -10]
+    line_xMax = [24, 24, 24, 24, 24, 24, 50]
+    line_dx = [1, 1, 1, 1, 1, 1, 5]
+    line_yMin = [-30, -30, 0, 0, -4, 0, 0]
+    line_yMax = [40, 40, 60, 60, 5, 4, 500]
+
+    for i in range(len(comDat)):
+        data = comDat[i]
+        if i < 7:
+            for j in range(len(barPlot_variable)):
+                if data['data_set'] == barPlot_variable[j]:
+                    bar_compare(data, barPlot_yLabel[j], barPlot_yMin[j], barPlot_yMax[j])
+        else:
+            for k in range(len(line_variable)):
+                if data['data_set'] == line_variable[k]:
+                    plot_lines(data, line_xLable[k], line_yLable[k], line_xMin[k], line_xMax[k], line_dx[k], line_yMin[k], line_yMax[k])
+# --------------------------------------------------------------------------------------------------
+
+def _generate_html_table(dataSet, BESTestTable):
+    setName = dataSet['data_set']
+    fileName = "data_{}.txt".format(setName)
+    filePath = os.path.join(BESTestTable, fileName)
+    if os.path.exists(filePath):
+        os.remove(filePath)
+    data = dataSet['data'][0]
+    tools = dataSet['tools']
+    with open(filePath, 'w') as f:
+        for i in range(len(data['firstCol'])):
+            f.write("<tr>" + os.linesep)
+            f.write("<td>{}</td>".format(data['firstCol'][i]) + os.linesep)
+            for j in range(len(tools)):
+                tool = tools[j]
+                f.write("<td>{}</td>".format(data[tool][i]) + os.linesep)
+                if 'peak_' in setName or 'max_' in setName or 'min_' in setName:
+                    toolHour = '{}_hour'.format(tool)
+                    f.write("<td>{}</td>".format(data[toolHour][i]) + os.linesep)
+            f.write("</tr>" + os.linesep)
+
+
+if __name__=="__main__":
+
+    if not POST_PROCESS_ONLY:
```

Comment:
This would be clear if you use something like
```
if DO_SIMULATIONS:
```
rather than negate a "only" command.

Reply:
Updated.